TACC/tup-ui
# Add ability for staff users to impersonate

## Overview:
Add an impersonation feature for staff users.

## Related:

- [TUP-441](https://jira.tacc.utexas.edu/browse/TUP-441)

## Changes:
- Add backend route for impersonating a user

## Testing:

1. Update `secrets.py` with a `TUP_SERVICES_ADMIN_JWT` setting ("TUP UI secrets.py for local dev" Stache entry)
2. Make sure you are logged in as a staff user.
3. In the top nav dropdown, select "impersonate user"
4. Enter someone's username and check that you see their version of the dashboard.

## UI:
<img width="1011" alt="image" src="https://user-images.githubusercontent.com/12601812/224113535-fe1b3a57-2faa-4ff7-8eb1-7e0b773e0848.png">


## Notes:
